src/store/user/provider.tsx

```diff
@@ -11,8 +11,7 @@ export const LoginProvider: React.FunctionComponent<{ children: React.ReactNode
     useEffect(() => {
         const token = localStorage.getItem("token");
```

Comment:
If it's used only once, thats good. But i'll still extract this to the method and place it in utils. But this method, should contain argument - saying what value you wanna get from localStorage. Then create another method which is returning that specific value i.e:
`getDataFromLocalStorage = (itemName: string) => { return localStorage.getItem(itemName)};`
and then use it like that:
`getTokenFromLocalStorage = () => { return getDataFromLocalStorage('token'); }`
What do you think about this?
That technique is called Partial Application